tasks/files-to-granules/schemas/input.json

```diff
@@ -0,0 +1,14 @@
+{
+  "title": "MoveGranulesInput",
+  "description": "Describes the input expected by the files-to-granules task",
+  "type": "array",
+  "items": {
+    "granules": {
```

Comment:
This doesn't seem to match the input in `tests/data/payload.json`.

Reply:
Interesting, I just copy-pasted from move-granules. You're completely right though, strange that validation didn't catch any part of that. Anyway, cbf1c2a70